### PR TITLE
update_settings_descriptions

### PR DESCRIPTION
**Pull request by libbyhopfauf**
This includes the addition of the description for "Unfiltered" mode and screenshot of edit settings GUI.

### CODE REVIEW

File: Resources/Documentation/settings.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Run edit mode by typing:
 ```
 vrecord -e
 ```
+(https://drive.google.com/open?id=1BY6Mu2qkBg1l7oETNBsFNK8Qs_hU6f-k)
```
Comment (dericed) [inline Resources/Documentation/settings.md]: Were you trying to insert an image or a link?

Comment (privatezero) [reply Resources/Documentation/settings.md]: Updated to image - I think resources should be kept as local as possible

Comment (libbyhopfauf) [reply Resources/Documentation/settings.md]: my bad! Yes, it was supposed to be an image.

Comment (libbyhopfauf) [reply Resources/Documentation/settings.md]: Screen Shot 2020-01-21 at 10 59 52 AM

**Pull request merged**